nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-ui/src/main/webapp/js/nf/canvas/nf-flow-version.js

```diff
@@ -206,7 +206,8 @@
                     registries.push({
                         text: registry.name,
                         value: registry.id,
-                        description: nfCommon.escapeHtml(registry.description)
+                        description: nfCommon.escapeHtml(registry.description),
+                        disabled: registry.validationStatus !== 'VALID' // TODO: verify this logic is correct
```

Comment:
This looks correct but we should check with @simonbence or @markap14 who are a little more familiar with the backend effort [1].

[1] #6433